darwin/cli.py

```diff
@@ -22,61 +35,77 @@ def main():
 
     # Authenticate user
     if args.command == "authenticate":
-        email = input("Username (email address): ")
-        password = getpass.getpass(prompt="Password: ", stream=None)
-        projects_dir = prompt("Project directory", "~/.darwin/projects")
-        f.authenticate(email, password, projects_dir)
-        print("Authentication succeeded. ")
+        api_key = getpass.getpass(prompt="API key: ", stream=None)
+        if api_key.strip() == "":
+            print(
+                "API Key needed, generate one for your team: https://darwin.v7labs.com/?settings=api-keys"
+            )
+            return
+        default_team = input("Make this team default? [y/N] ") in ["Y", "y"]
```

Comment:
As per discussion, I leave a note: make the call first and prompt the team name to the user to make it more intuitive